recipes/server.rb

```diff
@@ -0,0 +1,57 @@
+#
+# Cookbook Name:: strongdm
+# Recipe:: server
+#
+# Copyright © 2018 Applause App Quality, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+include_recipe 'strongdm::default'
+
+execute 'sdm-login-with-admin-token' do
+  command './sdm login'
+  cwd Chef::Config['file_cache_path']
+  environment('SDM_ADMIN_TOKEN' => node['strongdm']['admin_token'])
+  creates '/root/.sdm/auth.json'
+end
+
+execute 'sdm-admin-add-servers' do
+  command "./sdm admin servers add -p #{node['fqdn']} #{node['strongdm']['user']}@#{node['ipaddress']}"
+  cwd Chef::Config['file_cache_path']
+  environment('SDM_ADMIN_TOKEN' => node['strongdm']['admin_token'])
+  not_if "#{Chef::Config['file_cache_path']}/sdm admin servers list | grep #{node['fqdn']}"
```

Comment:
Why use the file cache path explicitly here when calling `sdm`? Does the `cwd` param not apply to the `not_if` command? Is this a "better safe than sorry" thing?

Reply:
Better safe than sorry. I couldn't find the answer in the docs and didn't feel like trial by fire.